pkg/annotations/ingress_annotations_test.go

```diff
@@ -340,6 +341,20 @@ var _ = Describe("Test ingress annotation functions", func() {
 		})
 	})
 
+	Context("test OverrideBackendHostName", func() {
+		It("returns error when ingress has no annotations", func() {
+			ing := &networking.Ingress{}
+			actual, err := OverrideBackendHostName(ing)
+			Expect(err).To(HaveOccurred())
+			Expect(actual).To(Equal(""))
+		})
+		It("returns the hostname", func() {
```

Comment:
This test name is incorrect
```suggestion
		It("returns true with correct annotation", func() {
```